src/renderer/index.html

```diff
@@ -9,43 +9,127 @@
 </head>
 <body>
     <div class="app-container">
-        <!-- Sidebar -->
-        <aside class="sidebar" id="sidebar">
-            <button class="sidebar-toggle" id="sidebar-toggle" title="Toggle sidebar">
-                <span class="toggle-icon">◀</span>
-            </button>
-            <div class="sidebar-header">
-                <h1 class="app-title">PowerPlatform<br>ToolBox</h1>
-            </div>
-            <nav class="sidebar-nav">
-                <button class="nav-item active" data-view="tools">
-                    <span class="nav-icon">🔧</span>
-                    <span class="nav-label">Tools</span>
+        <!-- Activity Bar (VSCode-like narrow sidebar) -->
+        <aside class="activity-bar" id="activity-bar">
```

Comment:
If I click the same option again then it toggles the expand of the sidebar. For eg. If I am on installed tool and the sidebar is expanded; I click Installed tools again it should collapse the sidebar so that the main content gets more space